charts/agent/values.schema.json

```diff
@@ -23,6 +24,100 @@
           ]
         }
       }
+    },
+    "localForwarder": {
+      "type": "object",
+      "required": [
+        "enabled",
+        "integrations",
+        "transmitMessageTypes"
+      ],
+      "if": {
```

Comment:
FYI: Not sure if the "if" here is supported on old version of `helm` CLI

Reply:
It appears to be...

I've been able to successfully use this change with helm 3.6.3 and 3.10.1. I can look into some other approaches here if you'd like to remove the if/then flow though.